### PR TITLE
NEPT-1546: Use an Attributes container for handling template attributes.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

**Comment by voidtek** [comment]
:+1:

File: templates/easy_breadcrumb/easy-breadcrumb.tpl.php [hunk 1]
```diff
@@ -5,8 +5,8 @@
  * Contains template file.
  */
 ?>
-<nav<?php print $attributes; ?>>
-  <span class="ecl-sr-only"><?php print render($here); ?></span>
+<nav<?php print $atomium['attributes']['wrapper']; ?>>
+  <span class="ecl-u-sr-only"><?php print render($here); ?></span>
```
Comment (Medi4tor) [inline templates/easy_breadcrumb/easy-breadcrumb.tpl.php]: Hi,
Are you sure of the name of this class: ecl-u-sr-only ?
K.R.

Comment (drupol) [reply templates/easy_breadcrumb/easy-breadcrumb.tpl.php]: I don't know but it's out of scope of this PR.

Comment (voidtek) [reply templates/easy_breadcrumb/easy-breadcrumb.tpl.php]: The ecl-u-sr-only is the new name of "sr-only" class.
see: https://github.com/ec-europa/europa-component-library/blob/f8f802859d3644bee0c473e8900869b993ed2874/framework/utilities/ecl-u-screen-reader/ecl-u-screen-reader.scss#L6

Comment (Medi4tor) [reply templates/easy_breadcrumb/easy-breadcrumb.tpl.php]: Ok good

**Comment by Medi4tor** [comment]
Good work in general, I see that some  components has been updated.
Nevertheless Travis failed on  pager component.
@drupol did you notice that?

File: .travis.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@ before_script:
   # Test latest commit on current branch.
   - git clone --branch=$TRAVIS_BRANCH https://github.com/$TRAVIS_REPO_SLUG.git sites/all/themes/contrib/ec_europa
   # And ec_europa theme should be based on the good branch of atomium.
+  - rm -fr sites/all/themes/contrib/atomium
```
Comment (Medi4tor) [inline .travis.yml]: These 2 lines could be changed when the atomium attributes-as-objects branch will be merged in 7.1.x

**Pull request merged**